app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -199,9 +198,7 @@ class Core(
                 RecordingDevicesMiddleware(context)
             )
 
-        if (!newMediaSessionApi) {
-            middlewareList.add(MediaMiddleware(context, MediaService::class.java))
-        }
+        middlewareList.add(MediaMiddleware(context, MediaService::class.java))
```

Comment:
Do we still need this? We were using it when we _weren't_ using the new API?

Reply:
oops you're right this was a mistake.  I'll fix it.  Thanks!